.lycheeignore

```diff
@@ -1,3 +1,5 @@
 # List of URLs to ignore in the link checker
 https://www.ghacks.net/*
 https://web.archive.org/web/*
+https://www.paypal.com/en_AT/i/scr/pixel.gif
+https://www.paypal.com/cgi-bin/webscr
```

Comment:
```suggestion
```

In my opinion, this should be committed if you think we're close to merge this PR (because here there is no longer a contribute.md file).

Reply:
I have nothing more to add to this pull request at the moment.